### ISSUE TITLE
Feature to pause a running job.

### ISSUE THREAD

**Issue by seczoid** [hashicorp/nomad#19899]
Hi, Is there any existing feature that would allow me to pause a running job and then resume it on demand.

**Comment by lgfa29** [comment]
Hi @seczoid 👋 

Thank you for the suggestion 🙂 

Could clarify what you mean by "pausing the job"? Would it be something like the [`docker container pause`](https://docs.docker.com/engine/reference/commandline/container_pause/) command?

**Comment by seczoid** [comment]
> Hi @seczoid 👋
> 
> Thank you for the suggestion 🙂
> 
> Could clarify what you mean by "pausing the job"? Would it be something like the [`docker container pause`](https://docs.docker.com/engine/reference/commandline/container_pause/) command?

yeah something like `docker container pause` and in the case of a raw_exec something like `SIGSTOP` to pause and `SIGCONT` to resume.
